lib/widgets/twake_link_text.dart

```diff
@@ -23,6 +26,26 @@ class TwakeLinkText extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final String? firstValidLink = _getFirstValidLink(text);
+
+    if (firstValidLink == null) {
```

Comment:
you don't have to put condition check inside build method, you should it check it upper and redirect it to another widget, because this is very hard to read